### ISSUE TITLE
Audio doesnt resume, when hold and Accept PSTN call and switching back to TwilioVoice call iOS

### ISSUE THREAD

**Issue by Kishorekumar-Elanchezhiyan** [twilio/cocoapod-specs#134]
We are using TwilioVoice (2.0.3) with CallKit
When we are in Voice call and a PSTN call comes in
I hold and accept the PSTN Call
 didDeactivate audioSession  gets called, Thats where we disable sdk audio by TwilioVoice.isAudioEnabled = false
then I ends the PSTN call in local device, didActivate audioSession gets called. There we enable audio by TwilioVoice.isAudioEnabled = true

But audio doesnt comeback.

Discussing this issue with @bchen-twilio, please refer https://github.com/twilio/cocoapod-specs/issues/96. I am yet to get a resolution

**Comment by Kishorekumar-Elanchezhiyan** [comment]
please refer https://github.com/twilio/cocoapod-specs/issues/96

**Comment by Kishorekumar-Elanchezhiyan** [comment]
Yes @bchen-twilio  it works with 3.0 SDK but as it is still under development we will not be able to ship it with our app. Our app is ready for production and we cannot ship with sdk under development.

**Comment by Kishorekumar-Elanchezhiyan** [comment]
@bchen-twilio When are we planning to release the TwilioVoice 3.x version. Please let me know if you have a roadmap

**Comment by bchen-twilio** [comment]
Hi @Kishorekumar-Elanchezhiyan 

Thanks for trying it out, and we definitely understand the requirement of production applications. The Voice iOS 3.0 SDK is currently in `preview2` and we plan to ship 2 beta releases in the coming weeks before we can claim GA quality. Based on the dev progress, we should be able to reach GA roughly between mid and end of September.

Again we really appreciate your patience and we will be working our best to ship this awesome next-gen SDK.

-bobie

**Comment by Kishorekumar-Elanchezhiyan** [comment]
Thanks @bchen-twilio
